client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -2,43 +2,32 @@
  * External dependencies
  */
 import React from 'react';
-import { connect } from 'react-redux';
-import { isMobile } from '@automattic/viewport';
 
 /**
  * Internal dependencies
  */
 import GoMobile from 'my-sites/customer-home/cards/features/go-mobile';
 import ChecklistSiteSetup from 'my-sites/customer-home/cards/primary/checklist-site-setup';
 import QuickLinks from 'my-sites/customer-home/cards/primary/quick-links';
-import getSiteChecklist from 'state/selectors/get-site-checklist';
-import isEligibleForDotcomChecklist from 'state/selectors/is-eligible-for-dotcom-checklist';
-import isSiteChecklistComplete from 'state/selectors/is-site-checklist-complete';
-import { getSelectedSiteId } from 'state/ui/selectors';
 
-const Primary = ( { checklistMode, displayChecklist } ) => {
-	return (
-		<>
-			{ // "Go Mobile" is displayed on the primary location when viewed in smaller viewports, so folks
-			// can see it on their phone without needing to scroll.
-			isMobile() && <GoMobile /> }
-			{ displayChecklist && <ChecklistSiteSetup checklistMode={ checklistMode } /> }
-			{ ! displayChecklist && <QuickLinks /> }
-		</>
-	);
+const cardComponents = {
+	'feature.go-mobile-phones': GoMobile,
```

Comment:
We'll need to update these identifies once D41010-code lands.

Reply:
4273d501800f5eec9a434a4b9d00227cd22eb086